db/records.py

```diff
@@ -17,11 +17,17 @@ class GroupFieldNotFound(FieldNotFound):
     pass
 
 
-def _get_primary_key_column(table):
-    primary_key_list = list(inspect(table).primary_key)
-    # We do not support getting by composite primary keys
-    assert len(primary_key_list) == 1
-    return primary_key_list[0]
+class NotUniquePrimaryKey(Exception):
+    pass
+
+
+def _get_primary_key_columns(table):
+    return list(inspect(table).primary_key)
+
+
+def _get_primary_key_column(table, idx):
+    primary_key_columns = _get_primary_key_columns(table)
```

Comment:
Please leave the `assert` in here so that calling this function if there are multiple primary keys fails.

Reply:
Actually, I think it would be better to move the `if len(primary_key_columns) != 1:` exception raising into this function so that you don't have to repeat it in multiple functions.